### DISCUSSION TITLE
Reaction and Comment Button

### DISCUSSION THREAD

**Discussion by samrit2442** [quarto-dev/quarto-cli#4875] [Q&A] [answered]
I want to add a reaction button, subscribe and comment section to my quarto blog like the screenshot below.
![image](https://user-images.githubusercontent.com/48163065/226049656-f2f2bf64-139f-430c-99c7-68e3645b8ff6.png)

Plz help me how can I do that?

**Comment by mcanouil** [comment, marked as answer]
For subscription: <https://quarto.org/docs/websites/website-blog.html#subscriptions>
For comments: <https://quarto.org/docs/reference/projects/websites.html#comments>

If none of these suit your needs, then you have to look for HTML/JavaScript that does on the Internet, and inject that code using the `include-*` keys/options: <https://quarto.org/docs/reference/formats/html.html#includes>.

**Reply by kurianbenoy** [reply]
Isn't he looking specifically for reactions? How to add that?

**Reply by mcanouil** [reply]
As I say, you can find plenty of examples/implementations on the Internet.
For example, a simple search gave me <https://codepen.io/inesmontani/pen/OyOEWg>.

**Comment by samrit2442** [comment]
Thank you all for your suggestions.

**Reply by kurianbenoy** [reply]
@samrit2442 if you are looking for reactions, maybe check giscuss for commenting. That's how I implemented reactions in my quarto website.

https://quarto.org/docs/output-formats/html-basics.html#commenting

**Reply by mcanouil** [reply]
@kurianbenoy FYI, It's the link I provided for "comments".

**Reply by kurianbenoy** [reply]
Thanks for pointing out. Given your link, I implemented giscuss in my blog. I wished I could specifically point to that section, yet it's not possible.
